Find a functionally equivalent code:android.os.Bundle.putChar:COMMENT
Method Modifier: public      
Comment:/**
 * Inserts a char value into the mapping of this Bundle, replacing
 * any existing value for the given key.
 *
 * @param key a String, or null
 * @param value a char, or null
 */

@@ -1,4 +1,3 @@
 {
-    unparcel();
-    mMap.put(key, value);
+    super.putChar(key, value);
 }

